.github/workflows/create-release.yml

```diff
@@ -0,0 +1,48 @@
+# This workflow automates release creation and builds and deploys to DockerHub
+name: Automates release gets latest tag, creates release, builds and push to DockerHub with latest tag
```

Comment:
Name can be little simpler: `Release project` and the above comment is already explaining its working.